### ISSUE TITLE
Dockerfile Operator Snapshot

### ISSUE THREAD

**Issue by kaslin** [oracle/weblogic-kubernetes-operator#79]
In the Dockerfile, where is the following file supposed to come from?

COPY target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar /operator/weblogic-kubernetes-operator.jar

This file does not appear to be included in the repository.

**Comment by markxnelson** [comment]
You need to build first, this is documented in the installation.md file.

**Comment by kaslin** [comment]
I ran this, from installation.md:

docker build -t weblogic-kubernetes-operator:developer --no-cache=true .

And got it failed with this output:

Step 5/8 : COPY target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar /operator/weblogic-kubernetes-operator.jar
COPY failed: stat /var/lib/docker/tmp/docker-builder223468053/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar: no such file or directory

I also don't have a /var/lib/docker/tmp directory (running this on a mac):
cd /var/lib/docker/tmp/
-bash: cd: /var/lib/docker/tmp/: No such file or directory

**Comment by markxnelson** [comment]
Do you have JDK 1.8 installed?

**Comment by markxnelson** [comment]
It works fine for me with JDK 1.8 and Maven installed, also on a Mac:

```
$ git clone https://github.com/oracle/weblogic-kubernetes-operator
Cloning into 'weblogic-kubernetes-operator'...
remote: Counting objects: 1721, done.
remote: Compressing objects: 100% (103/103), done.
remote: Total 1721 (delta 61), reused 57 (delta 16), pack-reused 1593
Receiving objects: 100% (1721/1721), 5.68 MiB | 10.21 MiB/s, done.
Resolving deltas: 100% (1036/1036), done.
$ java -version
java version "1.8.0_91"
Java(TM) SE Runtime Environment (build 1.8.0_91-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.91-b14, mixed mode)
$ mvn clean install
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building weblogic-kubernetes-operator 0.1.0-alpha-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ weblogic-kubernetes-operator ---
[INFO] Deleting /private/tmp/79/weblogic-kubernetes-operator/target
[INFO]
[INFO] --- build-helper-maven-plugin:3.0.0:add-source (add-source) @ weblogic-kubernetes-operator ---
[INFO] Source directory: /private/tmp/79/weblogic-kubernetes-operator/src-generated-swagger/main/java added.
[INFO] Source directory: /private/tmp/79/weblogic-kubernetes-operator/src/main/java added.
[INFO] Source directory: /private/tmp/79/weblogic-kubernetes-operator/src/main/java/../../build/java added.
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ weblogic-kubernetes-operator ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ weblogic-kubernetes-operator ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 107 source files to /private/tmp/79/weblogic-kubernetes-operator/target/classes
[INFO] /private/tmp/79/weblogic-kubernetes-operator/src/main/java/oracle/kubernetes/operator/rest/RestServer.java: /private/tmp/79/weblogic-kubernetes-operator/src/main/java/oracle/kubernetes/operator/rest/RestServer.java uses or overrides a deprecated API.
[INFO] /private/tmp/79/weblogic-kubernetes-operator/src/main/java/oracle/kubernetes/operator/rest/RestServer.java: Recompile with -Xlint:deprecation for details.
[INFO] /private/tmp/79/weblogic-kubernetes-operator/src/main/java/oracle/kubernetes/operator/rest/BaseDebugLoggingFilter.java: Some input files use unchecked or unsafe operations.
[INFO] /private/tmp/79/weblogic-kubernetes-operator/src/main/java/oracle/kubernetes/operator/rest/BaseDebugLoggingFilter.java: Recompile with -Xlint:unchecked for details.
[INFO]
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ weblogic-kubernetes-operator ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /private/tmp/79/weblogic-kubernetes-operator/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.7.0:testCompile (default-testCompile) @ weblogic-kubernetes-operator ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 18 source files to /private/tmp/79/weblogic-kubernetes-operator/target/test-classes
[INFO] /private/tmp/79/weblogic-kubernetes-operator/src/test/java/oracle/kubernetes/custom/TestCustomObjectApis.java: /private/tmp/79/weblogic-kubernetes-operator/src/test/java/oracle/kubernetes/custom/TestCustomObjectApis.java uses unchecked or unsafe operations.
[INFO] /private/tmp/79/weblogic-kubernetes-operator/src/test/java/oracle/kubernetes/custom/TestCustomObjectApis.java: Recompile with -Xlint:unchecked for details.
[INFO]
[INFO] --- maven-surefire-plugin:2.20:test (default-test) @ weblogic-kubernetes-operator ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:2.6:jar (default-jar) @ weblogic-kubernetes-operator ---
[INFO] Building jar: /private/tmp/79/weblogic-kubernetes-operator/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar
[INFO]
[INFO] --- maven-jar-plugin:2.6:jar (default) @ weblogic-kubernetes-operator ---
[INFO]
[INFO] --- maven-jar-plugin:2.6:test-jar (default) @ weblogic-kubernetes-operator ---
[INFO] Building jar: /private/tmp/79/weblogic-kubernetes-operator/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT-tests.jar
[INFO]
[INFO] --- maven-dependency-plugin:2.8:copy-dependencies (default) @ weblogic-kubernetes-operator ---
[INFO] Copying log4j-1.2.17.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/log4j-1.2.17.jar
[INFO] Copying animal-sniffer-annotations-1.14.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/animal-sniffer-annotations-1.14.jar
[INFO] Copying javax.ws.rs-api-2.1.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javax.ws.rs-api-2.1.jar
[INFO] Copying okio-1.6.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/okio-1.6.0.jar
[INFO] Copying gson-2.6.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/gson-2.6.2.jar
[INFO] Copying commons-lang3-3.7.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/commons-lang3-3.7.jar
[INFO] Copying swagger-annotations-1.5.15.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/swagger-annotations-1.5.15.jar
[INFO] Copying jsonp-jaxrs-1.1.1.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jsonp-jaxrs-1.1.1.jar
[INFO] Copying jackson-core-2.9.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-core-2.9.2.jar
[INFO] Copying jersey-common-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-common-2.26.jar
[INFO] Copying grizzly-framework-2.4.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/grizzly-framework-2.4.0.jar
[INFO] Copying jersey-server-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-server-2.26.jar
[INFO] Copying jersey-hk2-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-hk2-2.26.jar
[INFO] Copying javassist-3.22.0-CR2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javassist-3.22.0-CR2.jar
[INFO] Copying mimepull-1.9.6.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/mimepull-1.9.6.jar
[INFO] Copying okhttp-ws-2.7.5.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/okhttp-ws-2.7.5.jar
[INFO] Copying grizzly-http-2.4.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/grizzly-http-2.4.0.jar
[INFO] Copying client-java-1.0.0-beta2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/client-java-1.0.0-beta2.jar
[INFO] Copying jaxb-api-2.3.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jaxb-api-2.3.0.jar
[INFO] Copying client-java-api-1.0.0-beta2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/client-java-api-1.0.0-beta2.jar
[INFO] Copying hamcrest-core-1.3.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/hamcrest-core-1.3.jar
[INFO] Copying jersey-entity-filtering-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-entity-filtering-2.26.jar
[INFO] Copying commons-codec-1.9.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/commons-codec-1.9.jar
[INFO] Copying grizzly-http-server-2.4.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/grizzly-http-server-2.4.0.jar
[INFO] Copying hk2-api-2.5.0-b42.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/hk2-api-2.5.0-b42.jar
[INFO] Copying snakeyaml-1.18.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/snakeyaml-1.18.jar
[INFO] Copying validation-api-1.1.0.Final.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/validation-api-1.1.0.Final.jar
[INFO] Copying jersey-media-jaxb-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-media-jaxb-2.26.jar
[INFO] Copying jackson-dataformat-yaml-2.9.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-dataformat-yaml-2.9.2.jar
[INFO] Copying joda-time-2.9.3.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/joda-time-2.9.3.jar
[INFO] Copying jersey-media-json-processing-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-media-json-processing-2.26.jar
[INFO] Copying javaee-web-api-7.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javaee-web-api-7.0.jar
[INFO] Copying aopalliance-repackaged-2.5.0-b42.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/aopalliance-repackaged-2.5.0-b42.jar
[INFO] Copying jersey-client-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-client-2.26.jar
[INFO] Copying hk2-utils-2.5.0-b42.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/hk2-utils-2.5.0-b42.jar
[INFO] Copying jsr305-1.3.9.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jsr305-1.3.9.jar
[INFO] Copying jackson-databind-2.9.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-databind-2.9.2.jar
[INFO] Copying j2objc-annotations-1.1.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/j2objc-annotations-1.1.jar
[INFO] Copying hk2-locator-2.5.0-b42.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/hk2-locator-2.5.0-b42.jar
[INFO] Copying protobuf-java-3.4.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/protobuf-java-3.4.0.jar
[INFO] Copying jackson-annotations-2.9.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-annotations-2.9.2.jar
[INFO] Copying javax.inject-2.5.0-b42.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javax.inject-2.5.0-b42.jar
[INFO] Copying error_prone_annotations-2.0.18.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/error_prone_annotations-2.0.18.jar
[INFO] Copying system-rules-1.16.1.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/system-rules-1.16.1.jar
[INFO] Copying javax.annotation-api-1.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javax.annotation-api-1.2.jar
[INFO] Copying jackson-module-jaxb-annotations-2.8.4.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-module-jaxb-annotations-2.8.4.jar
[INFO] Copying guava-22.0.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/guava-22.0.jar
[INFO] Copying javax.inject-1.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javax.inject-1.jar
[INFO] Copying osgi-resource-locator-1.0.1.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/osgi-resource-locator-1.0.1.jar
[INFO] Copying logging-interceptor-2.7.5.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/logging-interceptor-2.7.5.jar
[INFO] Copying javax.json-api-1.1.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javax.json-api-1.1.2.jar
[INFO] Copying junit-4.12.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/junit-4.12.jar
[INFO] Copying okhttp-2.7.5.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/okhttp-2.7.5.jar
[INFO] Copying jersey-container-grizzly2-http-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-container-grizzly2-http-2.26.jar
[INFO] Copying javax.json-1.1.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/javax.json-1.1.2.jar
[INFO] Copying jersey-media-json-jackson-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-media-json-jackson-2.26.jar
[INFO] Copying jackson-datatype-joda-2.9.2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-datatype-joda-2.9.2.jar
[INFO] Copying jersey-media-multipart-2.26.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jersey-media-multipart-2.26.jar
[INFO] Copying jackson-core-asl-1.9.11.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/jackson-core-asl-1.9.11.jar
[INFO] Copying client-java-proto-1.0.0-beta2.jar to /private/tmp/79/weblogic-kubernetes-operator/target/lib/client-java-proto-1.0.0-beta2.jar
[INFO]
[INFO] --- maven-shade-plugin:2.4.3:shade (default) @ weblogic-kubernetes-operator ---
[INFO] Including io.swagger:swagger-annotations:jar:1.5.15 in the shaded jar.
[INFO] Including org.glassfish.jersey.core:jersey-client:jar:2.26 in the shaded jar.
[INFO] Including javax.ws.rs:javax.ws.rs-api:jar:2.1 in the shaded jar.
[INFO] Including org.glassfish.jersey.core:jersey-common:jar:2.26 in the shaded jar.
[INFO] Including javax.annotation:javax.annotation-api:jar:1.2 in the shaded jar.
[INFO] Including org.glassfish.hk2:osgi-resource-locator:jar:1.0.1 in the shaded jar.
[INFO] Including org.glassfish.hk2.external:javax.inject:jar:2.5.0-b42 in the shaded jar.
[INFO] Including org.glassfish.jersey.media:jersey-media-multipart:jar:2.26 in the shaded jar.
[INFO] Including org.jvnet.mimepull:mimepull:jar:1.9.6 in the shaded jar.
[INFO] Including org.glassfish.jersey.media:jersey-media-json-jackson:jar:2.26 in the shaded jar.
[INFO] Including org.glassfish.jersey.ext:jersey-entity-filtering:jar:2.26 in the shaded jar.
[INFO] Including com.fasterxml.jackson.module:jackson-module-jaxb-annotations:jar:2.8.4 in the shaded jar.
[INFO] Including org.glassfish.jersey.media:jersey-media-json-processing:jar:2.26 in the shaded jar.
[INFO] Including org.glassfish:jsonp-jaxrs:jar:1.1.1 in the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-core:jar:2.9.2 in the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-annotations:jar:2.9.2 in the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-databind:jar:2.9.2 in the shaded jar.
[INFO] Including com.fasterxml.jackson.datatype:jackson-datatype-joda:jar:2.9.2 in the shaded jar.
[INFO] Including org.glassfish:javax.json:jar:1.1.2 in the shaded jar.
[INFO] Including javax.json:javax.json-api:jar:1.1.2 in the shaded jar.
[INFO] Including joda-time:joda-time:jar:2.9.3 in the shaded jar.
[INFO] Including org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar:2.26 in the shaded jar.
[INFO] Including org.glassfish.grizzly:grizzly-http-server:jar:2.4.0 in the shaded jar.
[INFO] Including org.glassfish.grizzly:grizzly-http:jar:2.4.0 in the shaded jar.
[INFO] Including org.glassfish.grizzly:grizzly-framework:jar:2.4.0 in the shaded jar.
[INFO] Including org.glassfish.jersey.core:jersey-server:jar:2.26 in the shaded jar.
[INFO] Including org.glassfish.jersey.media:jersey-media-jaxb:jar:2.26 in the shaded jar.
[INFO] Including javax.validation:validation-api:jar:1.1.0.Final in the shaded jar.
[INFO] Including org.glassfish.jersey.inject:jersey-hk2:jar:2.26 in the shaded jar.
[INFO] Including org.glassfish.hk2:hk2-locator:jar:2.5.0-b42 in the shaded jar.
[INFO] Including org.glassfish.hk2.external:aopalliance-repackaged:jar:2.5.0-b42 in the shaded jar.
[INFO] Including org.glassfish.hk2:hk2-api:jar:2.5.0-b42 in the shaded jar.
[INFO] Including javax.inject:javax.inject:jar:1 in the shaded jar.
[INFO] Including org.glassfish.hk2:hk2-utils:jar:2.5.0-b42 in the shaded jar.
[INFO] Including org.javassist:javassist:jar:3.22.0-CR2 in the shaded jar.
[INFO] Including com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.2 in the shaded jar.
[INFO] Including org.yaml:snakeyaml:jar:1.18 in the shaded jar.
[INFO] Including commons-codec:commons-codec:jar:1.9 in the shaded jar.
[INFO] Including org.codehaus.jackson:jackson-core-asl:jar:1.9.11 in the shaded jar.
[INFO] Including io.kubernetes:client-java:jar:1.0.0-beta2 in the shaded jar.
[INFO] Including io.kubernetes:client-java-api:jar:1.0.0-beta2 in the shaded jar.
[INFO] Including com.squareup.okhttp:okhttp:jar:2.7.5 in the shaded jar.
[INFO] Including com.squareup.okio:okio:jar:1.6.0 in the shaded jar.
[INFO] Including com.squareup.okhttp:logging-interceptor:jar:2.7.5 in the shaded jar.
[INFO] Including com.google.code.gson:gson:jar:2.6.2 in the shaded jar.
[INFO] Including io.kubernetes:client-java-proto:jar:1.0.0-beta2 in the shaded jar.
[INFO] Including org.apache.commons:commons-lang3:jar:3.7 in the shaded jar.
[INFO] Including com.squareup.okhttp:okhttp-ws:jar:2.7.5 in the shaded jar.
[INFO] Including com.google.guava:guava:jar:22.0 in the shaded jar.
[INFO] Including com.google.code.findbugs:jsr305:jar:1.3.9 in the shaded jar.
[INFO] Including com.google.errorprone:error_prone_annotations:jar:2.0.18 in the shaded jar.
[INFO] Including com.google.j2objc:j2objc-annotations:jar:1.1 in the shaded jar.
[INFO] Including org.codehaus.mojo:animal-sniffer-annotations:jar:1.14 in the shaded jar.
[INFO] Including log4j:log4j:jar:1.2.17 in the shaded jar.
[INFO] Including com.github.stefanbirkner:system-rules:jar:1.16.1 in the shaded jar.
[INFO] Including com.google.protobuf:protobuf-java:jar:3.4.0 in the shaded jar.
[INFO] Including javax:javaee-web-api:jar:7.0 in the shaded jar.
[INFO] Including javax.xml.bind:jaxb-api:jar:2.3.0 in the shaded jar.
[INFO] Including junit:junit:jar:4.12 in the shaded jar.
[INFO] Including org.hamcrest:hamcrest-core:jar:1.3 in the shaded jar.
[WARNING] javax.annotation-api-1.2.jar, javaee-web-api-7.0.jar define 15 overlapping classes:
[WARNING]   - javax.annotation.ManagedBean
[WARNING]   - javax.annotation.PreDestroy
[WARNING]   - javax.annotation.Resource$AuthenticationType
[WARNING]   - javax.annotation.Generated
[WARNING]   - javax.annotation.security.DeclareRoles
[WARNING]   - javax.annotation.Priority
[WARNING]   - javax.annotation.Resource
[WARNING]   - javax.annotation.security.DenyAll
[WARNING]   - javax.annotation.sql.DataSourceDefinitions
[WARNING]   - javax.annotation.security.RunAs
[WARNING]   - 5 more...
[WARNING] validation-api-1.1.0.Final.jar, javaee-web-api-7.0.jar define 106 overlapping classes:
[WARNING]   - javax.validation.ConstraintDeclarationException
[WARNING]   - javax.validation.GroupDefinitionException
[WARNING]   - javax.validation.ConstraintValidatorContext$ConstraintViolationBuilder$LeafNodeBuilderDefinedContext
[WARNING]   - javax.validation.metadata.ParameterDescriptor
[WARNING]   - javax.validation.Validator
[WARNING]   - javax.validation.ConstraintValidatorContext$ConstraintViolationBuilder
[WARNING]   - javax.validation.constraints.AssertTrue$List
[WARNING]   - javax.validation.constraints.Past
[WARNING]   - javax.validation.metadata.GroupConversionDescriptor
[WARNING]   - javax.validation.constraints.Max
[WARNING]   - 96 more...
[WARNING] javax.inject-1.jar, javax.inject-2.5.0-b42.jar, javaee-web-api-7.0.jar define 6 overlapping classes:
[WARNING]   - javax.inject.Inject
[WARNING]   - javax.inject.Singleton
[WARNING]   - javax.inject.Scope
[WARNING]   - javax.inject.Named
[WARNING]   - javax.inject.Provider
[WARNING]   - javax.inject.Qualifier
[WARNING] javax.json-api-1.1.2.jar, javax.json-1.1.2.jar, jaxb-api-2.3.0.jar define 1 overlapping classes:
[WARNING]   - module-info
[WARNING] javax.json-api-1.1.2.jar, javaee-web-api-7.0.jar define 25 overlapping classes:
[WARNING]   - javax.json.stream.JsonParser
[WARNING]   - javax.json.stream.JsonParserFactory
[WARNING]   - javax.json.JsonString
[WARNING]   - javax.json.JsonValue
[WARNING]   - javax.json.Json
[WARNING]   - javax.json.stream.JsonLocation
[WARNING]   - javax.json.stream.JsonParsingException
[WARNING]   - javax.json.stream.JsonParser$Event
   22 # Installation
[WARNING]   - javax.json.JsonWriter
[WARNING]   - javax.json.JsonValue$ValueType
[WARNING]   - 15 more...
[WARNING] javax.ws.rs-api-2.1.jar, javaee-web-api-7.0.jar define 123 overlapping classes:
[WARNING]   - javax.ws.rs.ext.RuntimeDelegate$HeaderDelegate
[WARNING]   - javax.ws.rs.DefaultValue
[WARNING]   - javax.ws.rs.core.StreamingOutput
[WARNING]   - javax.ws.rs.HEAD
[WARNING]   - javax.ws.rs.ext.WriterInterceptor
[WARNING]   - javax.ws.rs.core.Request
[WARNING]   - javax.ws.rs.ext.Providers
[WARNING]   - javax.ws.rs.container.Suspended
[WARNING]   - javax.ws.rs.container.ConnectionCallback
[WARNING]   - javax.ws.rs.client.FactoryFinder
[WARNING]   - 113 more...
[WARNING] maven-shade-plugin has detected that some class files are
[WARNING] present in two or more JARs. When this happens, only one
[WARNING] single version of the class is copied to the uber jar.
[WARNING] Usually this is not harmful and you can skip these warnings,
[WARNING] otherwise try to manually exclude artifacts based on
[WARNING] mvn dependency:tree -Ddetail=true and the above output.
[WARNING] See http://maven.apache.org/plugins/maven-shade-plugin/
[INFO] Replacing original artifact with shaded artifact.
[INFO] Replacing /private/tmp/79/weblogic-kubernetes-operator/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar with /private/tmp/79/weblogic-kubernetes-operator/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT-shaded.jar
[INFO] Dependency-reduced POM written at: /private/tmp/79/weblogic-kubernetes-operator/dependency-reduced-pom.xml
[INFO]
[INFO] --- exec-maven-plugin:1.6.0:java (default) @ weblogic-kubernetes-operator ---
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ weblogic-kubernetes-operator ---
[INFO] Installing /private/tmp/79/weblogic-kubernetes-operator/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar to /Users/marnelso/.m2/repository/oracle/kubernetes/weblogic-kubernetes-operator/0.1.0-alpha-SNAPSHOT/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar
[INFO] Installing /private/tmp/79/weblogic-kubernetes-operator/dependency-reduced-pom.xml to /Users/marnelso/.m2/repository/oracle/kubernetes/weblogic-kubernetes-operator/0.1.0-alpha-SNAPSHOT/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.pom
[INFO] Installing /private/tmp/79/weblogic-kubernetes-operator/target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT-tests.jar to /Users/marnelso/.m2/repository/oracle/kubernetes/weblogic-kubernetes-operator/0.1.0-alpha-SNAPSHOT/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT-tests.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 37.785 s
[INFO] Finished at: 2018-02-09T16:43:52-05:00
[INFO] Final Memory: 45M/494M
[INFO] ------------------------------------------------------------------------
$ docker build -t weblogic-kubernetes-operator:developer --no-cache=true .
Sending build context to Docker daemon  1.763GB
Step 1/8 : FROM store/oracle/serverjre:8
 ---> c5e638a9290e
Step 2/8 : RUN mkdir /operator
 ---> Running in 44355eca1565
Removing intermediate container 44355eca1565
 ---> d35c1619d56a
Step 3/8 : ENV PATH=$PATH:/operator
 ---> Running in 4369f3851e61
Removing intermediate container 4369f3851e61
 ---> f2fddc09d607
Step 4/8 : COPY src/main/scripts/* /operator/
 ---> dc57140120a9
Step 5/8 : COPY target/weblogic-kubernetes-operator-0.1.0-alpha-SNAPSHOT.jar /operator/weblogic-kubernetes-operator.jar
 ---> 7fb9fba654a7
Step 6/8 : HEALTHCHECK --interval=1m --timeout=10s   CMD /operator/livenessProbe.sh
 ---> Running in d3c8ed481d1f
Removing intermediate container d3c8ed481d1f
 ---> 32a6007255e2
Step 7/8 : WORKDIR /operator/
Removing intermediate container c532669a3e98
 ---> bd6828c1ff4c
Step 8/8 : CMD ["operator.sh"]
 ---> Running in 17d24265b203
Removing intermediate container 17d24265b203
 ---> f23f6c695285
Successfully built f23f6c695285
Successfully tagged weblogic-kubernetes-operator:developer
$ docker images | grep developer
weblogic-kubernetes-operator                                            developer                                         f23f6c695285        44 seconds ago      302MB
```

**Comment by kaslin** [comment]
I have jdk, but not maven. Wasn't aware I needed either of those. I only installed jdk because it had a pop up saying it needed it. These pre-reqs should be stated clearly.

**Comment by markxnelson** [comment]
I added those requirements to the documentation.  Apologies for the inconvenience.  The intention was to publish the image and just let people pull it from a registry, but we are still working through the process to do that, so for now, and probably for the next few weeks, we need people to build the image themselves.  When we changed the documentation from pulling the image to building it, I missed some details.
